_includes/tutorials/kafka-streams-schedule-operations/kstreams/code/build.gradle

```diff
@@ -33,16 +33,16 @@ apply plugin: "com.github.johnrengelman.shadow"
 dependencies {
     implementation "org.apache.avro:avro:1.10.1"
     implementation "org.slf4j:slf4j-simple:1.7.30"
-    implementation "org.apache.kafka:kafka-streams:2.6.0"
-    implementation('io.confluent:kafka-streams-avro-serde:5.5.1') {
+    implementation "org.apache.kafka:kafka-streams:2.7.0"
+    implementation('io.confluent:kafka-streams-avro-serde:6.1.0') {
         // this transitive dependency exclusion
-        // is needed as kafka-clients:5.5.1-ccs has a version
+        // is needed as kafka-clients:6.1.0-ccs has a version
```

Comment:
I doubt this is the cause of the failure.  The unit tests pass but the test-harness is the one that's failing.  It's strange because it seemed to pass in previous runs.